static/src/js/components/OrderStatus/__tests__/mocks.js

```diff
@@ -74,3 +82,83 @@ export const orderStatusPropsTwo = {
   }
 }
 
+export const orderStatusPropsEsi = {
+  authToken: 'testToken2',
+  order: {
```

Comment:
I'm confused with the payload here -- the top level key here is `order` that would suggest the data within in is one order, but the next key is `collections`, which has `esi_orders` inside of it. Based on the data that looks like its a `retrieval_collection` object from the database. `access_method` inside of that object contains the order data, but that data is not stored in the database, and the plan is to leave that unchanged. If we need to discuss this I can make some time this afternoon.

Reply:
I've basically mimicked what I saw in 1.0 for this ticket. I was under the impression any improvements to the payloads would be happening when we hook this up to real order responses. I can try and do some of that now, but I think we should probably chat about what we can make better first.

Reply:
This will be handled under the order status work